bundles/org.openhab.binding.siemenshvac/src/main/resources/OH-INF/thing/ozw672.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="siemenshvac"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Bridge Thing Type -->
+	<bridge-type id="ozw672">
+		<label>OZW672 IP Gateway</label>
+		<description>This is a OZW672 IP interface</description>
+
+		<config-description>
+			<parameter name="baseUrl" type="text">
+				<label>Base URL</label>
+				<description>The URL of the Siemens Hvac IP gateway</description>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="userName" type="text">
+				<description>User name of the Siemens Hvac gateway</description>
+				<required>false</required>
+				<label>userName</label>
+				<default>Administrator</default>
+			</parameter>
+			<parameter name="userPassword" type="text">
+				<description>User password of the Siemens Hvac gateway</description>
+				<required>false</required>
+				<label>userPassword</label>
+				<default>password</default>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+
+
+
+
+	<channel-type id="outside-temp">
+		<item-type>Number:Temperature</item-type>
+		<label>Outside temp</label>
```

Comment:
```suggestion
		<label>Outside Temperature</label>
```

See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions.